static/app/views/settings/project/projectTeams.spec.tsx

```diff
@@ -8,12 +8,13 @@ import {
 } from 'sentry-test/reactTestingLibrary';
 
 import TeamStore from 'sentry/stores/teamStore';
+import {Organization, Project} from 'sentry/types';
```

Comment:
we should really `import type {...` these imports. Hopefully we can enable ts strictTypeImports and auto fix this soon. It drops the entire import and doesn't end up evaluating all the modules inside sentry/types

Reply:
yes it would be nice if this was done automatically when saving the document :)